eth/api_tracer_oe.go

```diff
@@ -50,7 +50,7 @@ type TraceRewardAction struct {
 	RewardType string          `json:"rewardType,omitempty"`
 }
 
-// setConfigTracerToOpenEthereum forces the Tracer to the OpenEthereum one
+// setConfigTracerToOpenEthereum forces the Tracer to the OpenEthereum one, mutating the argument value.
 func setConfigTracerToOpenEthereum(config *TraceConfig) *TraceConfig {
 	if config == nil {
```

Comment:
@meowsbits when `config == nil`, there is no actual pointer to assign to. In this case the config is being scoped in `setConfigTracerToOpenEthereum`. Assigning to an indirect pointer, can't work in this case either. If we want to do it without assigning from the caller func, then we have to call it like:

```go
if config == nil {
	config = new(TraceConfig)
}
setConfigTracerToOpenEthereum(config)
```

Reply:
@meowsbits what do you think on this? any better ideas?

Reply:
Reverted for now, so as it works on external review. Will leave this review unresolved though.